MVC/controller.js

```diff
@@ -11,12 +16,28 @@ exports. getAPI = (req, res, next) => {
     res.status(200).send(fetchAPI())
 }
 
-exports. getArticle = (req, res, next) => {
+exports. getArticleById = (req, res, next) => {
     const { article_id } = req.params
-    fetchArticle(article_id)
+    fetchArticleById(article_id)
     .then((article) => {
         res.status(200).send({article: article})
     })
     .catch(next)
     // this catches BOTH SQL errors and custom ones
+}
+
+exports. getArticles = (req, res, next) => {
+    fetchArticles()
+    .then(({ articles }) => {
+        res.status(200).send({articles: articles})
+    })
+    // add comment count from comments data to that article id
```

Comment:
Lets remove all comments and console.logs before committing to keep our code tidy